### PR TITLE
Fogl 3189: Minor improvements on 1.7.0

### PR DESCRIPTION
**Pull request by sstefano69**

### CODE REVIEW

**Comment by sstefano69** [comment]
> Let's make it:
> 
> ```
> 
> 	if [[ $os_name == *"Raspbian"*  ]]; then
> 
> 		cpulimit -l 40 -- python3 -m foglamp.tasks.statistics "$@"
> 	else
> 		# Standard execution on other platforms
> 		python3 -m foglamp.tasks.statistics "$@"
> 	fi
> ```
> 
> i.e. remove the redundant complexity of Redhat checks and nested if

done, thanks for the good suggestion

**Comment by sstefano69** [comment]
> Also remove line
> `os_version=`(grep -o '^VERSION_ID=.*' /etc/os-release | cut -f2 -d" | sed 's/"/`

done, thanks for the good suggestion

File: scripts/foglamp [hunk 1]
```diff
@@ -485,7 +485,7 @@ foglamp_status() {
 
                 # Show Tasks
                 foglamp_log "info" "=== FogLAMP tasks:" "outonly" "pretty"
-                ps -ef | grep -o 'python3 -m foglamp.tasks.*' | grep -o 'foglamp.tasks.*' | grep -v 'foglamp.tasks\.\*' || true
+                ps -ef | grep -v 'cpulimit*' | grep -o 'python3 -m foglamp.tasks.*' | grep -o 'foglamp.tasks.*'    | grep -v 'foglamp.tasks\.\*' || true
```
Comment (praveen-garg) [inline scripts/foglamp]: Are we sure, we want to invert-match?

Comment (sstefano69) [reply scripts/foglamp]: yes, we want all the rows apart from the ones elated to cpulimit

Comment (praveen-garg) [reply scripts/foglamp]: So we shall stop showing stats collector task  in foglamp status on Raspbian now? Would not it be better to `| sed 's/cpulimit -l 40 --//g'` and retain the current behavior!

File: scripts/tasks/statistics [hunk 2]
```diff
@@ -18,11 +18,11 @@ fi
 cd "${FOGLAMP_ROOT}/python"
 
 os_name=`(grep -o '^NAME=.*' /etc/os-release | cut -f2 -d\" | sed 's/"//g')`
-os_version=`(grep -o '^VERSION_ID=.*' /etc/os-release | cut -f2 -d\" | sed 's/"//g')`
 
-if [[ ( $os_name == *"Red Hat"* || $os_name == *"CentOS"* ) &&  $os_version == *"7"* ]]; then
+if [[ $os_name == *"Raspbian"*  ]]; then
```
Comment (praveen-garg) [inline scripts/tasks/statistics]: With this, we can remove `cpulimit` from control files for aarch64 and x86_64 https://github.com/foglamp/foglamp-pkg/tree/develop/packages/Debian. 

Also I'm not sure, we should replace the check based on arch=`uname -m` Or arch=`arch`?!

**Pull request merged**